.gitignore

```diff
@@ -10,6 +10,9 @@
 # Virtualenv
 /.venv/
 
+# Virtualenv
+/.venv/
```

Comment:
Are these lines the repetition of the lines above?

Reply:
May be some other developer also commit the same content to master.